<?xml version="1.0"?>
		
<launch>

	<!-- Arguments for led driver -->
	<arg name = "robot_model" default = "robot"/>
	<arg name = "leds_port" default = "/dev/ttyUSB_LEDS"/>
	<arg name = "exclusive_signals" default= "false"/>

	<!-- Arguments for user application example -->
	<arg name="cmd_vel_topic" default = "robotnik_base_control/cmd_vel"/>
	<arg name="verbose" default="false" />

	<!-- Launch user application example-->
	<node pkg="robotnik_leds_example" type="$(arg robot_model)_leds.py" name="$(arg robot_model)_robot_node" output="screen">
		<param name="cmd_vel_topic" value="$(arg cmd_vel_topic)"/>
		<param name="verbose" value="$(arg verbose)" />
	</node>

	<!-- Launch led driver -->
	<include file="$(find robotnik_signal_leds_manager)/launch/leds_manager.launch">
		<arg name="robot_model" value="$(arg robot_model)"/>
		<arg name="leds_port" value="$(arg leds_port)"/>
		<arg name="exclusive_signals" value = "$(arg exclusive_signals)"/>
	</include>
 
</launch>
